### PR TITLE
draft for the first blocks

### PR DESCRIPTION
**Pull request by innabelaya**

### CODE REVIEW

File: common.blocks/attach/__button/attach__button.en.title.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+A button as a block’s element
```
Comment (persidskiy) [inline common.blocks/attach/__button/attach__button.en.title.txt]: Why do we use the `*.title.txt` files for each block and element, while we have same description in `block.md` file?
I think, if we remove this files, the repository will be more purely.

/cc @veged @narqo

Comment (veged) [inline common.blocks/attach/__button/attach__button.en.title.txt]: i think so — we considering to auto extract title from markdown header

Comment (persidskiy) [inline common.blocks/attach/__button/attach__button.en.title.txt]: Great! By the way, we will have all info about block and his elemens and modifiers in one file. This staff won't be scattered by folders.

File: common.blocks/attach/attach.en.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+* Theme of a button: native representation of a button by the browser.  
+* Localization: ru.
+* Button's message text (`__button`): «Выбрать файл»  
+* Text message rendered with no file selected (`__no-file`): "Файл не выбран"
+
+Following example demonstrates this:
+
+```` 
+    {
```
Comment (persidskiy) [inline common.blocks/attach/attach.en.md:35]: Why do you use indent? There is no need.

File: common.blocks/attach/attach.en.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+If theme is not spesified (declared), the block applies only default values:  
+
+* Theme of a button: native representation of a button by the browser.  
+* Localization: ru.
+* Button's message text (`__button`): «Выбрать файл»  
+* Text message rendered with no file selected (`__no-file`): "Файл не выбран"
+
+Following example demonstrates this:
+
+````
```
Comment (persidskiy) [inline common.blocks/attach/attach.en.md]: You may use ```js for better look of code.

File: common.blocks/attach/__button/attach__button.en.title.txt (hunk 1 above)
Comment (innabelaya) [inline common.blocks/attach/__button/attach__button.en.title.txt]: thanx! I'll remove them all

File: common.blocks/attach/attach.en.md (hunk 3 above)
Comment (innabelaya) [inline common.blocks/attach/attach.en.md]: thanx)

**Pull request closed without merging**